source/documentation/disaster_recovery_guide/task/task_CreateAnsiblePlaybooks.adoc

```diff
@@ -13,9 +13,12 @@ Ansible is used to initiate and manage the disaster recovery failover and failba
 ** Networks with the same general connectivity as the primary site.
 * Replicated storage. See <<storage_considerations_active-passive>> for more information.
 +
-NOTE: The replicated storage that contains virtual machines and templates must not be attached to the secondary site.
+[NOTE]
```

Comment:
Again, old style had an icon and a "NOTE" word.
The new style is better / more common?